.babelrc

```diff
@@ -1,14 +1,19 @@
 {
   "presets": [
-    ["es2015", { "loose": true }],
-    "stage-1",
+    ["env", {
+      "targets": {
+        "ie": 9,
+        "uglify": true
+      }
+    }],
     "react"
   ],
   "plugins": [
+    "transform-class-properties",
+    "transform-object-rest-spread",
+    "transform-export-extensions",
```

Comment:
maybe just use stage 1?

Reply:
stage 1 is less maintainable, plus adds everything 1 -4 negating most of the env preset